rpcs3/Emu/RSX/GL/GLHelpers.h

```diff
@@ -1251,7 +1263,7 @@ namespace gl
 			}
 		};
 
-		GLuint m_id = GL_NONE;
+		GLuint m_id = 0;
```

Comment:
But GL_NONE exists as an enumeration for this purpose so why remove it?

Reply:
Previous usage was mixed up, I believe GL_NONE is for `glDrawBuffer`/`glDeleteBuffers` but was being used in some other functions, should we use GL_NONE for all `m_id` that are zero, or only in what functions should GL_NONE be used to represent zero?

Reply:
We're supposed to use NULL but since we don't use that in "modern c++" we use GL_NONE instead to preserve readability. Unlike vulkan, OGL does not have GL_NULL_HANDLE and uses GL_NONE for this purpose.